### PR TITLE
Dp 20703 font preload

### PR DESCRIPTION
**Pull request by joeg8612**
**Description:**
Changed font preloads to use woff2 formats.

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-20703

**To Test:**
- [ ] Load home page and inspect with network view. Verify that woff2 fonts are loaded but woff fonts are not loaded. There should be 2 font files loaded rather than 4.

**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by weitzman** [comment]
FYI Drupal's new theme uses a similar technical and adds a . `type="font/woff2"` attribute. Not sure how important that is. See https://www.drupal.org/files/issues/2020-10-29/3176910-13.patch

**Comment by tom-fleming** [comment]
@weitzman I think this is probably the best explanation I've found: https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content#Including_a_MIME_type. 

TL;DR - Setting the MIME type helps with preloading by allowing the browser to determine support first. 

With that said, pretty much all modern browsers support woff2, so I'm not too concerned that we're not including that, but it might help with some older browsers that don't support the format like IE11, though in my own test, the woff2 font did not load, which is what we want. IE11 must have another way to determine that it shouldn't load the font.

**Comment by tom-fleming** [comment]
I will add though that it looks like in IE11 we are no longer falling back on an EOT-format font and are instead falling back on probably Helvetica. Either way, no fonts are loaded in IE11 with this change.

**Comment by tkoleary** [comment]
@tom-fleming `"Noto Sans","Helvetica",Arial, sans-serif;` are the fallbacks in the CSS `font-family` attribute but there are also fallbacks in the @font-face rules for "Noto Sans VF" and "Noto Sans Italic VF"
